cmd/loadtest/loadtest.go

```diff
@@ -1100,12 +869,23 @@ func loadtestCall(ctx context.Context, c *ethclient.Client, nonce uint64, ltCont
 	tops = configureTransactOpts(tops)
 
 	t1 = time.Now()
-	_, err = contracts.CallLoadTestFunctionByOpCode(f, ltContract, tops, *iterations)
-	t2 = time.Now()
+	defer func() { t2 = time.Now() }()
+	if *ltp.CallOnly {
+		tops.NoSend = true
+		var tx *ethtypes.Transaction
+		tx, err = contracts.CallLoadTestFunctionByOpCode(f, ltContract, tops, *iterations)
+		if err != nil {
+			return
+		}
+		msg := txToCallMsg(tx)
+		_, err = c.CallContract(ctx, msg, nil)
+	} else {
+		_, err = contracts.CallLoadTestFunctionByOpCode(f, ltContract, tops, *iterations)
+	}
 	return
 }
 
-func loadtestCallPrecompiledContracts(ctx context.Context, c *ethclient.Client, nonce uint64, ltContract *contracts.LoadTester, useSelectedAddress bool) (t1 time.Time, t2 time.Time, err error) {
+func loadTestCallPrecompiledContracts(ctx context.Context, c *ethclient.Client, nonce uint64, ltContract *contracts.LoadTester, useSelectedAddress bool) (t1 time.Time, t2 time.Time, err error) {
```

Comment:
seeing this again reminds me of the discussion of boolean flags haha

Reply:
Yeah. Sometimes it's ugly but the easiest way to get it done. I didn't look into this one specifically.